From af4af632c2f23e7de2a36bde0b0171ffb87a1446 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 14 Feb 2018 07:59:53 +0100
Subject: [PATCH 1/1] cmd: scsi: move CMD_SCSI to device access commands

Configuration option CMD_SCSI should be placed in the same submenu as
CMD_SATA and CMD_MMC.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/Kconfig | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/cmd/Kconfig b/cmd/Kconfig
index 7c517493983..885712f00f3 100644
--- a/cmd/Kconfig
+++ b/cmd/Kconfig
@@ -877,6 +877,15 @@ config CMD_SAVES
 	  Provides a way to save a binary file using the Motorola S-Record
 	  format over the serial line.
 
+config CMD_SCSI
+	bool "scsi - Access to SCSI devices"
+	default y if SCSI
+	help
+	  This provides a 'scsi' command which provides access to SCSI (Small
+	  Computer System Interface) devices. The command provides a way to
+	  scan the bus, reset the bus, read and write data and get information
+	  about devices.
+
 config CMD_SDRAM
 	bool "sdram - Print SDRAM configuration information"
 	help
@@ -1466,15 +1475,6 @@ config CMD_REISER
 	    reiserls - list files
 	    reiserload - load a file
 
-config CMD_SCSI
-	bool "scsi - Access to SCSI devices"
-	default y if SCSI
-	help
-	  This provides a 'scsi' command which provides access to SCSI (Small
-	  Computer System Interface) devices. The command provides a way to
-	  scan the bus, reset the bus, read and write data and get information
-	  about devices.
-
 config CMD_YAFFS2
 	bool "yaffs2 - Access of YAFFS2 filesystem"
 	depends on YAFFS2
-- 
2.15.1

